internal/pkg/warewulfd/parser.go

```diff
@@ -23,7 +24,9 @@ type parserInfo struct {
 
 func parseReq(req *http.Request) (parserInfo, error) {
 	var ret parserInfo
-
+    var ipaddr []string
+    var ipaddrtemp string
```

Comment:
I suggest that these variable declarations should be made implicitly with `:=` when they are used below.